src/renderer/components/dock/__test__/dock-tabs.test.tsx

```diff
@@ -8,6 +8,20 @@ import { createResourceTab } from "../create-resource.store";
 import { createTerminalTab } from "../terminal.store";
 import { observable } from "mobx";
 
+Object.defineProperty(window, "matchMedia", {
```

Comment:
This is the actual fix for unhandled exceptions, right?

Reply:
To some of them. I was also getting "KubeApi.parseApi is not a function" which is why there are those mocks in "kube-api-parse.test.ts".

Reply:
This test is fixed without needed to mock `matchMedia`.
https://github.com/lensapp/lens/pull/2037. But we can remeber such cases where cleaning test file isn't enough.

Reply:
Again, this should be removed for now. Test passes without these changes ^